### ISSUE TITLE
Facebook Pixel / Integration not included in analytics.js file

### ISSUE THREAD

**Issue by cgpsn** [thebarty/analytics-js-without-segment#4]
the analytics.js file, loaded from CDN (unminified for easier searching ) https://cdnjs.com/libraries/analytics.js/2.9.1 only defines a Facebook Conversion Tracking integration.

How do I go about loading other open source integrations? https://github.com/segmentio/analytics.js-integrations/tree/master/integrations/facebook-pixel

**Comment by thebarty** [comment]
Good question. Have you found out how to do it?

**Comment by cgpsn** [comment]
@thebarty negatory, abandoned the project. You're supposed to be able to compile your own with only the integrations you want though I could never get it to compile correctly. We've now moved to single app calls with service distribution via tag manager.

**Comment by thebarty** [comment]
@caseygipson thanks for the quick feedback!

Seems like segment is making it hard to compile the latest analytics.js version.

Anyone able to do this?

Pull requests welcome!

**Comment by omgwtfwow** [comment]
@thebarty @caseygipson yeah I worked it out for a project a few months ago, look here  https://github.com/omgwtfwow/analytics.js
